Scripts/DBotPredictOutOfTheBox/DBotPredictOutOfTheBox.py

```diff
@@ -0,0 +1,85 @@
+# pylint: disable=no-member
+import demisto_ml
+
+from CommonServerPython import *
+
+OUT_OF_THE_BOX_MODEL_NAME = 'demisto_out_of_the_box_model'
+OUT_OF_THE_BOX_MODEL_PATH = '/var/oob_model.ftz'
+EVALUATION_PATH = '/var/oob_evaluation.json'
+HASH_SEED = 5381
+
+
+def oob_model_exists():
+    res_model = demisto.executeCommand("getMLModel", {"modelName": OUT_OF_THE_BOX_MODEL_NAME})[0]
+    return not is_error(res_model)
+
+
+def load_oob_model():
+    with open(OUT_OF_THE_BOX_MODEL_PATH, 'rb') as input_file:
+        encryped_model = input_file.read()
+    encoded_model = demisto_ml.decrypt_model(encryped_model)
+    res = demisto.executeCommand('createMLModel', {'modelData': encoded_model,
+                                                   'modelName': OUT_OF_THE_BOX_MODEL_NAME,
+                                                   'modelLabels': ['legit', 'spam', 'malicious'],
+                                                   'modelOverride': 'true'})
+    if is_error(res):
+        return_error(get_error(res))
+
+    with open(EVALUATION_PATH, 'r') as json_file:
```

Comment:
I think it's better to change this logic, and do everything in the docker image - create function `get_oob_model()` that will load and decrypt and return the model